updateStableIds/src/main/java/org/reactome/release/updateStableIds/UpdateStableIds.java

```diff
@@ -0,0 +1,171 @@
+		// 'excluded' classes were Regulation classes, the filter was not needed anymore.
+//		final List<String> acceptedClassesWithStableIdentifiers = Arrays.asList("Pathway","SimpleEntity","OtherEntity","DefinedSet","Complex","EntityWithAccessionedSequence","GenomeEncodedEntity","Reaction","BlackBoxEvent","CandidateSet","Polymer","Depolymerisation","Polymerisation","Drug","FailedReaction","EntitySet");
+//		List<String> excludedClassesWithStableIdentifiers = Arrays.asList("PositiveGeneExpressionRegulation","PositiveRegulation","NegativeRegulation","Requirement","NegativeGeneExpressionRegulation");
+//		ResultSet classesWithStableIdentifiers = dbaGkCentral.executeQuery("SELECT DISTINCT _class FROM DatabaseObject WHERE StableIdentifier IS NOT NULL", null);		
+
+		logger.info("Fetching all Event and PhysicalEntity instances");
+		// Get all Event and PhysicalEntity instances and combine them into one large List
+		Collection<GKInstance> eventInstances = dbaSlice.fetchInstancesByClass(Event);
+		Collection<GKInstance> physicalEntityInstances = dbaSlice.fetchInstancesByClass(PhysicalEntity);
+		List<GKInstance> sliceInstances = new ArrayList<GKInstance>();
+		sliceInstances.addAll(eventInstances);
+		sliceInstances.addAll(physicalEntityInstances);
+
+		int incrementedCount = 0;
+		int notIncrementedCount = 0;
+		logger.info("Total instances to check: " + sliceInstances.size());
+
+		for (GKInstance sliceInstance : sliceInstances) {
+			logger.info("Checking " + sliceInstance);
+			GKInstance gkCentralInstance = dbaGkCentral.fetchInstance(sliceInstance.getDBID());
+			GKInstance prevSliceInstance = dbaPrevSlice.fetchInstance(sliceInstance.getDBID());
+			// Check if instance is new and that it exists on gkCentral (they could be deleted)
+			if (prevSliceInstance != null && gkCentralInstance != null) {
+				
+				// Compare number of 'Modified' instances between slices
+				Collection<GKInstance> sliceInstanceModified = sliceInstance.getAttributeValuesList(modified);
+				Collection<GKInstance> prevSliceInstanceModified = prevSliceInstance.getAttributeValuesList(modified);
+				if (sliceInstanceModified.size() > prevSliceInstanceModified.size()) {
+					// Make sure StableIdentifier instance exists
+					if (sliceInstance.getAttributeValue(stableIdentifier) != null && gkCentralInstance.getAttributeValue(stableIdentifier) != null) {
+						logger.info("\tIncrementing " + sliceInstance.getAttributeValue(stableIdentifier));
+						incrementStableIdentifier(sliceInstance, dbaSlice, sliceIE);
+						incrementStableIdentifier(gkCentralInstance, dbaGkCentral, gkCentralIE);
+						incrementedCount++;
+					} else if (sliceInstance.getAttributeValue(stableIdentifier) == null){
+						logger.warn(sliceInstance + ": could not locate StableIdentifier instance");
+					} else {
+						logger.warn(prevSliceInstance + ": Instance from previous slice did not have StableIdentifier instance");
```

Comment:
Can this be an info statement instead of a warn statement since this case can occur legitimately when a new instance exists that didn't exist in the previous release?

Reply:
Yes I can change it, I think at time of writing I didn't realize this wasn't a negative thing